doc/whats_new/v0.20.rst

```diff
@@ -422,6 +422,15 @@ Linear, kernelized and related models
   better for unscaled features. :issue:`8361` by :user:`Gaurav Dhingra <gxyd>`
   and :user:`Ting Neo <neokt>`.
 
+Feature Extraction
+
+- Deprecate ``stop_words='english'`` parameter in 
+  :class:`feature_extraction.text.CountVectorizer`, 
+  :class:`feature_extraction.text.TfidfVectorizer`, 
+  :class:`feature_extraction.text.HashingVectorizer` as the current list is 
+  broken, leading to unexpected results. :issue:`10735` 
```

Comment:
please also copy the reason for the deprecation to the doc

Reply:
Sure. A confirmation please, 

`Copy the reason for the deprecation to the doc`:

Places to copy:
1.  CountVec doc (http://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html)
2. Copy to HashVectectorizer doc
3. Copy to TFIDF doc

Any other places?

Reply:
That's enough from my side :)

Reply:
@qinhanmin2014 I am having trouble finding out how the above pages are generated. I grep searched but nothing helped. Could you please let me know where could I find these `.rst files`

Reply:
@shaz13 The CountVectorizer / HashingVectorizer docs are generated from from the corresponding docstrings. 
The TFIDF documentation can be modifed in https://github.com/scikit-learn/scikit-learn/blob/master/doc/modules/feature_extraction.rst